cibuildwheel/macos.py

```diff
@@ -94,8 +137,8 @@ def call(args, env=None, cwd=None, shell=False):
         call(['python', '--version'], env=env)
 
         # install pip & wheel
-        call(['python', get_pip_script, '--no-setuptools', '--no-wheel'], env=env, cwd="/tmp")
-        assert os.path.exists(os.path.join(installation_bin_path, 'pip'))
+        call(['python', get_pip_script], env=env, cwd="/tmp")
+        assert os.path.exists(os.path.join(installation_bin_path, 'pip')), os.listdir(installation_bin_path)
```

Comment:
This assertion message looks a little funky...

```suggestion
        assert os.path.exists(os.path.join(installation_bin_path, 'pip')), 'pip not found in installation_bin_path. It contains: {}'.format(os.listdir(installation_bin_path))
```

Reply:
(Although if the `pip` assert was going to be added to `make_symlinks`, we could remove it from here)

Reply:
Ah, still some debug output I forgot to remove! Good catch. Let's just remove the message; I think it should be fine without, as it was.

We can't check for `pip` before, though, because it might not be installed.